server/utils/queries/libraryItemsBookFilters.js

```diff
@@ -676,11 +676,14 @@ module.exports = {
       ],
       attributes: {
         include: [
-          [Sequelize.literal('(SELECT max(mp.updatedAt) FROM bookSeries bs, mediaProgresses mp WHERE mp.mediaItemId = bs.bookId AND mp.userId = :userId AND bs.seriesId = series.id)'), 'recent_progress']
+          [Sequelize.literal('(SELECT max(mp.updatedAt) FROM bookSeries bs, mediaProgresses mp WHERE mp.mediaItemId = bs.bookId AND mp.userId = :userId AND bs.seriesId = series.id)'), 'recent_progress'],
+          [Sequelize.literal('(SELECT CAST(max(bs.sequence) as FLOAT) FROM bookSeries bs, mediaProgresses mp WHERE mp.mediaItemId = bs.bookId AND mp.userId = :userId AND bs.seriesId = series.id)'), 'maxSequence'],
+          [Sequelize.literal('(SELECT json_extract(libraries.settings, "$.onlyShowLaterBooksInContinueSeries") FROM libraries WHERE id = :libraryId)'), 'onlyShowLaterBooksInContinueSeries']
```

Comment:
This is unnecessary because we can get the library settings from the function that calls this one. The library in `/objects/Library` is passed into the function that calls this one and has all the settings.

This way we can also check if that setting is set before adding the additional query.

Reply:
great catch! i 👍 

changed so that the entire library object is sent into the function instead, and that it will only make the maxSequence SQL query if the setting is turned on

also found a `var` that i used unconsciously (since I code way more golang than js…) that I changed to a let